com.unity.render-pipelines.high-definition/Tests/Editor/BuildProcessors/HDRPPreprocessShaderTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using UnityEditor;
+using UnityEditor.Build;
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.HighDefinition.Tests
+{
+    public class HDRPPreprocessShadersTests
+    {
+        [SetUp]
+        public void InitializeBuildPreprocessors()
+        {
+            // Try to initialize all build preprocessors
+            foreach (var preprocessor in TypeCache.GetTypesDerivedFrom<IPreprocessBuildWithReport>())
+            {
+                try
+                {
+                    var buildReport = Activator.CreateInstance(preprocessor) as IPreprocessBuildWithReport;
+                    buildReport.OnPreprocessBuild(default);
+                }
+                catch
+                {
+                }
```

Comment:
Why a Try-Catch discarding all error? If there is an error we will have wrong initialization without knowing it